src/llmq/quorums_chainlocks.cpp

```diff
@@ -35,17 +35,14 @@ std::string CChainLockSig::ToString() const
 
 CChainLocksHandler::CChainLocksHandler()
 {
-    scheduler = new CScheduler();
-    CScheduler::Function serviceLoop = boost::bind(&CScheduler::serviceQueue, scheduler);
-    scheduler_thread = new boost::thread(boost::bind(&TraceThread<CScheduler::Function>, "cl-schdlr", serviceLoop));
+    scheduler = std::make_unique<CScheduler>();
+    CScheduler::Function serviceLoop = std::bind(&CScheduler::serviceQueue, scheduler.get());
+    scheduler_thread = std::make_unique<std::thread>(std::bind(&TraceThread<CScheduler::Function>, "cl-schdlr", serviceLoop));
```

Comment:
this violates usage notes in `scheduler.h`, switching from boost to std for schedulers should be done in the corresponding backport imo

Reply:
reverted